tests/chainer_tests/functions_tests/pooling_tests/test_roi_pooling_2d.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestROIPooling2D(unittest.TestCase):
+
+    def setUp(self):
+        N = 4
+        n_channels = 3
+        self.x = numpy.random.randn(N, n_channels, 12, 8).astype(numpy.float32)
```

Comment:
Is this test stable? I mean, roi pooling contains `max` operator. It causes unstable behavior when values in an input variable contains similar values. Please check unit test of max-pooling.